### PR TITLE
V2.0.5

### PR DESCRIPTION
**Pull request by ocefpaf**
There is no PyPI release so I changed the source to GitHub.

### CODE REVIEW

**Comment by dopplershift** [comment]
Need some kind of work-around for libpng--the static library in the current libpng package was built with a different vs compiler. On my matplotlib testing build, this was accomplished by [copying the shared library over the static one](https://github.com/conda-forge/matplotlib-feedstock/pull/238).

File: recipe/bld.bat [hunk 1]
```diff
@@ -1,3 +1,12 @@
 SET GRIBAPI_DIR=%PREFIX%
 SET ZLIB_DIR=%PREFIX%
+SET PNG_DIR=%PREFIX%
+SET ZLIB_DIR=%PREFIX%
+
+:: Replace the static library with shared
+del %LIBRARY_LIB%\libpng16_static.lib
+copy %LIBRARY_LIB%\libpng16.lib %LIBRARY_LIB%\libpng16_static.lib
```
Comment (isuruf) [inline recipe/bld.bat:8]:
```suggestion
copy %LIBRARY_LIB%\libpng16.lib %LIBRARY_LIB%\png.lib
```

File: recipe/bld.bat [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+del %LIBRARY_LIB%\zlibstatic.lib
+copy %LIBRARY_LIB%\z.lib %LIBRARY_LIB%\zlibstatic.lib
+
 %PYTHON% -m pip install . --no-deps -vv
```
Comment (isuruf) [inline recipe/bld.bat:12]:
```suggestion
%PYTHON% -m pip install . --no-deps -vv
del %LIBRARY_LIB%\png.lib
```

File: recipe/bld.bat [hunk 3]
```diff
@@ -1,3 +1,12 @@
 SET GRIBAPI_DIR=%PREFIX%
 SET ZLIB_DIR=%PREFIX%
+SET PNG_DIR=%PREFIX%
+SET ZLIB_DIR=%PREFIX%
+
+:: Replace the static library with shared
+copy %LIBRARY_LIB%\libpng16.lib %LIBRARY_LIB%\png.lib
+del %LIBRARY_LIB%\zlibstatic.lib
+copy %LIBRARY_LIB%\z.lib %LIBRARY_LIB%\zlibstatic.lib
```
Comment (isuruf) [inline recipe/bld.bat:9]: I don't think these 2 lines are necessary

Comment (ocefpaf) [reply recipe/bld.bat:9]: I'll remove them in the next update. Forgot I set the automerge.

Thanks for the help!

**Pull request merged**